Frameworks/UIKit/UIDatePicker.mm

```diff
@@ -814,6 +813,8 @@ static void resetPickerPositions(UIDatePicker* self) {
  @Status Interoperable
 */
 - (void)setMinimumDate:(NSDate*)date {
+    [date retain];
+    [_minimumDate release];
     _minimumDate = date;
```

Comment:
Can we change these ivars to use StrongId? That seems like it would be more concise and avoid other issues like this in the future.